Update qa.m3u

trancer1994 opened the pull request.

trancer1994 committed:
Update qa.m3u

LaneSh4d0w commented:
Just take off the whiteline between your additions and the end of the playlist and I think it'll be good to go.

trancer1994 committed:
Update qa.m3u

This should be better. Then again, I tried removing the whiteline and must've removed the wrong bit because it affected something further up.
